content/docs/getting-started/edit-content.md

```diff
@@ -0,0 +1,123 @@
+
+//...
+
+const pageData = {
+  title: 'Tina is not a CMS',
+  body: 'It is a toolkit for creating a custom CMS.',
+}
+
+function PageContent() {
+  // 2. Define the form configuration object
+  const formConfig = {
+    id: 'tina-tutorial-index',
+    label: 'Edit Page',
+    fields: [
+      {
+        name: 'title',
+        label: 'Title',
+        component: 'text',
+      },
+      {
+        name: 'body',
+        label: 'Body',
+        component: 'textarea',
+      },
+    ],
+    initialValues: pageData,
+    onSubmit: async () => {
+      window.alert('Saved!')
+    },
+  }
+
+  // 3. Create the form
+  const [editableData, form] = useForm(formConfig)
+
+  // 4. Register it with the CMS
+  usePlugin(form)
+
+  return (
+    <section className="App-header">
+      <img src={logo} className="App-logo" alt="logo" />
+      {/**
+       * 5. Render the `editableData` returned from `useForm`
+       */}
+      <h1>{editableData.title}</h1>
+      <p>{editableData.body}</p>
+      <ExitButton />
+    </section>
+  )
+}
+
+//...
+```
+
+## Form Configuration
+
+`useForm` needs a [form configuration object](/docs/plugins/forms#form-configuration) with properties that determine how the form behaves on load and save, what fields are available, and other metadata.
+
+While there are other form config properties, the key ones to examine in the example above are `id`, `fields`, `initialValues`, and `onSubmit`. We'll look closely at each of these now.
+
+### _id_
+
+The `id` is unique identifier for the form. Since our example is the only instance of this form, the `id` is hard-coded. But if this form were in a template component, we would generate the `id` from the content to distinguish it from other form instances.
+
+### _fields_
+
+The `fields` array is comprised of [field definitions](/docs/plugins/fields#field-definition). All fields share a common [base configuration](docs/plugins/fields#field-config). Field definitions need at least two properties: a `name` or the path to the data and a `component` to edit that data from.
+
+You will be working with fields a lot in Tina. To get more familiar, try to adjust the `label` property or add a new field to the array. Reference [the documentation](/docs/plugins/fields) for a full list of default field plugins.
+
+### _initialValues_
+
+`initialValues` contains the data used to populate the form. If you do need to load data asynchronously on form creation, you'd use a function called `loadInitialValues` (we'll get to that soon).
+
+### _onSubmit_
+
+This is a function that runs when the form is saved. Right now it just gives an alert message, but we will use this function later to post changes to a backend.
+
+## Edit the data
+
+Head back to http://localhost:3000, enable the CMS, open the sidebar and try to update the content. You should now be able to edit the title and body copy on the demo!
+
+Next, we'll look at setting up a simple backend to retrieve and save data from a 3rd-party API.
```

Comment:
I wanted to do this right away. What if we move this above the "Form Configuration" section

* Make the change
* Look at the change
* Explain the change